config.py

```diff
@@ -0,0 +1,37 @@
+# Drivetrain
+front_left_move_id = 1
+front_left_turn_id = 2
+front_left_encoder_port = 1
+front_left_encoder_zeroed_pos = 0
+
+front_right_move_id = 3
+front_right_turn_id = 4
+front_right_encoder_port = 2
+front_right_encoder_zeroed_pos = 0
+
+back_left_move_id = 5
+back_left_turn_id = 6
+back_left_encoder_port = 3
+back_left_encoder_zeroed_pos = 0
+
+back_right_move_id = 7
+back_right_turn_id = 8
+back_right_encoder_port = 4
+back_right_encoder_zeroed_pos = 0
+
+driver_centric: bool = True
+drivetrain_reversed: bool = False
+
+# Gyro
+gyro_id = 0
+
+# Intake
+roller_id = 9
+hinge_id = 10
+intake_init_pos = 100
+intake_deploy_pos = 0
+intake_standard_pos = intake_init_pos
+intake_roller_speed = 1
+intake_idle_speed = 0
+intake_hinge_speed = 0.25
```

Comment:
No need for a speed. Operator should not be controlling the positions of the intake, use set positions.